conf/distro/include/rpb.inc

```diff
@@ -20,3 +20,5 @@ INHERIT += "rm_work"
 INHERIT += "buildhistory"
 INHERIT += "image-buildinfo"
 BUILDHISTORY_COMMIT = "1"
+
+EXTRA_OEMAKE += "-e MAKEFLAGS="
```

Comment:
See 5.9.4. section:
http://www.yoctoproject.org/docs/2.2/ref-manual/ref-manual.html

The recommended fix is to add this to recipes which depend on his.